README.md

```diff
@@ -18,6 +18,13 @@ build the source code, run the tests, and contribute your changes to the project
 
 ## How to install
 
+### macOS/Linux installation via Homebrew
+
+Thanks to [Efrat19](https://github.com/efrat19), arduino-cli is availabe as a brew formula since version `0.5.0`:
```

Comment:
Hi @Efrat19,
Thanks a lot for your contribution, we appreciate a lot a PR that enhances the ease of installation for the cli!

May I ask you to modify the line below according to the suggestion?

```suggestion
The arduino-cli is available as a brew formula since version `0.5.0`:
```

The `README` is still the main source of documentation for the project and we'd like to keep install instructions simple and direct. We still want to give our contributors more visibility, https://github.com/arduino/arduino-cli/pull/442  is a first step in that direction but we plan to do more and more as the project grows.

Thanks a lot for your contribution!